website/content/en/blog/log-namespacing.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Log Namespacing
+short: Log Namespacing
+description: Changing Vector's data model
+authors: ["fuchsnj"]
+date: "2023-??-??"
+badges:
+  type: announcement
+  domains: ["log namespacing"]
```

Comment:
Domains are definitely loosely defined, but maybe `data model` might be better here?